examples/todos-flow/src/containers/visible-todo-list.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import { connect } from 'react-redux';
+
+import { toggleTodo } from '../actions/todos';
+import { visibleTodosSelector } from '../selectors/todos';
+import TodoList from '../components/todo-list';
+
+import type { State, Dispatch } from '../types';
+import type { Connector } from 'react-redux';
+import type { Props } from '../components/todo-list';
+
+const mapStateToProps = (state: State) => {
+  return {
+    todos: visibleTodosSelector(state)
+  };
+};
+
+const mapDispatchToProps = (dispatch: Dispatch) => {
+  return {
+    onTodoClick: id => {
+      dispatch(toggleTodo(id));
+    }
+  };
+};
+
+const connector: Connector<{}, Props> = connect(
+  mapStateToProps,
+  mapDispatchToProps
+);
+
+export default connector(TodoList);
```

Comment:
This should be named `VisibleTodoList.js` and the displayName on the component should be`VisibleTodoList` as well.